commands/actions/publish.go

```diff
@@ -251,6 +253,8 @@ func mustBuildProject(sourcesDir string, tsconfig *typescript.TsConfig) {
 		return
 	}
 
+	util.RemoveDirWithContent(filepath.Join(sourcesDir, *tsconfig.CompilerOptions.OutDir))
```

Comment:
Are we sure we can delete this folder? Is there anything that the user can irreversibly lose when we do this?

Reply:
yes, that folder (`/dist`, `/out`, etc) contains just build output from the typescript compiler